src/SFA.DAS.LevyTransferMatching.Web/Views/Applications/GetApplications.cshtml

```diff
@@ -0,0 +1,75 @@
+﻿@using SFA.DAS.EmployerUrlHelper
```

Comment:
Can this view be called just Applications.cshtml instead?

Reply:
yeah, amended